ecf/hurricane/plots/jevs_hurricane_global_det_tcgen_plots.ecf

```diff
@@ -49,7 +49,7 @@ export cyc=00
 export job=jevs_hurricane_global_det_tcgen_plots_${cyc}
 
 #Define the directories of your TC genesis stats files
-export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/1.0/hurricane_global_det/tcgen/stats
+export COMINstats=/lfs/h2/emc/ptmp/$USER/com/evs/v1.0/hurricane_global_det/tcgen/stats
```

Comment:
Let's utilize the version settings in `run.ver`. Please use ${evs_ver} in place of "v1.0" in COMINstats definition.